tests/test_wave_03.py

```diff
@@ -55,12 +59,18 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
+    assert INTRIGUE_3 in friends_unique_movies
+    assert HORROR_1 in friends_unique_movies
+    assert FANTASY_4 in friends_unique_movies
```

Comment:
👍 Nice job with this assertion. 

Consider adding an assertion to check that the movie that was added to the first friend's watched list on line 54 was not added to Amanda's watched list.

```py 
assert INTRIGUE_3 not in amandas_data["watched"]